src/chapters/comm_brevity.tex

```diff
@@ -0,0 +1,260 @@
+\chapter{COMM BREVITY}
+\thumbtab{Comm Brevity}{9}
+\localtableofcontents
+\thispagestyle{plain}
+\cleardoublepage
+
+\section{A-A COMMUNICATION}
+
+\subsection{INTERCEPT}
+
+\begin{tcoloritemize}
+    \blueitem[ANGELS]
+    Height of FRIENDLY aircraft in thousands of feet from MSL.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, AT ANGELS 20.
+        \end{minipage}
+    \end{center}    
+
+    \blueitem[AZIMUTH]
+    1. A picture label describing the lateral separation of two GROUPS.\newline
```

Comment:
No reason to manually make a list with "1." etc. instead use `\begin{enumerate}` environment with `\item`s.
Additionally, you can then add a `\label{...}` which you can reference with a `\ref{...}` to automatically reference that item